diff --git a/source/kernel/linux-4.4.y/drivers/video/fbdev/Kconfig b/source/kernel/linux-4.4.y/drivers/video/fbdev/Kconfig
index f07a0974f..d6248077d 100644
--- a/source/kernel/linux-4.4.y/drivers/video/fbdev/Kconfig
+++ b/source/kernel/linux-4.4.y/drivers/video/fbdev/Kconfig
@@ -74,7 +74,7 @@ config FB_BOOT_VESA_SUPPORT
 config FB_CFB_FILLRECT
 	tristate
 	depends on FB
-	default n
+	default y
 	---help---
 	  Include the cfb_fillrect function for generic software rectangle
 	  filling. This is used by drivers that don't provide their own
@@ -83,7 +83,7 @@ config FB_CFB_FILLRECT
 config FB_CFB_COPYAREA
 	tristate
 	depends on FB
-	default n
+	default y
 	---help---
 	  Include the cfb_copyarea function for generic software area copying.
 	  This is used by drivers that don't provide their own (accelerated)
@@ -92,7 +92,7 @@ config FB_CFB_COPYAREA
 config FB_CFB_IMAGEBLIT
 	tristate
 	depends on FB
-	default n
+	default y
 	---help---
 	  Include the cfb_imageblit function for generic software image
 	  blitting. This is used by drivers that don't provide their own
